src/Servers/IIS/AspNetCoreModuleV2/CommonLib/HostFxrResolver.h

```diff
@@ -50,22 +50,17 @@ class HostFxrResolver
     );
 
     static
-    VOID
-    FindDotNetFolders(
-        const std::filesystem::path& path,
-        std::vector<std::wstring> & pvFolders
-    );
-
-    static
-    std::wstring
-    FindHighestDotNetVersion(
-        std::vector<std::wstring> & vFolders
-    );
+        BOOL
```

Comment:
I'd just use bool here... no reason for this to be BOOL (and same with the one above).

Please fix the indentation too.